wherehows-web/app/controllers/application.ts

```diff
@@ -45,6 +44,11 @@ export default class Application extends Controller {
   @service('banners')
   banners: BannerService;
 
+  /**
+   * Keyword of the current search to pass it down to the search bar
+   */
+  keyword: string;
```

Comment:
Confused as to why this is being stored on the application controller?

Reply:
Well, since the search bar is in the application.hbs, we need to pass keyword (which is the query), so we can restore the state correctly. Moving this component to a different location requires more refractor that I do not want to do at this moment